pkgs/tools/graphics/ldgallery/viewer/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, pkgs, nodejs-12_x, pandoc }:
+{ lib, stdenv, fetchFromGitHub, pkgs, nodejs-12_x, pandoc, darwin }:
 
 with lib;
 
```

Comment:
```suggestion
```

Reply:
It's also used at [line 33], not only [27], so that justifies the `with` a bit more :)

[line 33]: https://github.com/NixOS/nixpkgs/blob/a7d50d085a192921fef4d542affe53b088d339a6/pkgs/tools/graphics/ldgallery/viewer/default.nix#L33 
[27]: https://github.com/NixOS/nixpkgs/pull/123226/files#diff-40eb1f240d6bc8cee5c65cb249f6eb9289080c770de27cbecbc1e7f224ac40f7R27

Reply:
It should be used over the smallest scope possible and lib is usually not put into a with because it is short.